mailer.js

```diff
@@ -219,12 +251,15 @@ module.exports = function (log) {
       templateValues: {
         device: this._formatUserAgentInfo(message),
         email: message.email,
+        ip: message.ip,
         link: link,
+        location: this._constructLocationString(message),
         oneClickLink: oneClickLink,
         passwordChangeLink: this.createPasswordChangeLink(message.email),
         passwordChangeLinkAttributes: this._passwordChangeLinkAttributes(message.email),
         supportLinkAttributes: linkAttributes(this.supportUrl),
-        supportUrl: this.supportUrl
+        supportUrl: this.supportUrl,
+        timestamp: this._constructLocalTimeString(message.timeZone, message.acceptLanguage)
```

Comment:
Passing `message.acceptLanguage` does not work because `acceptLanguage` is a header that can contain several locales, e.g., `es;q=0.8,en-US;q=0.5,en;q=0.3`. When I tested with Spanish as my primary language and English as my secondary, this is the output I see is in English:

<img width="344" alt="screen shot 2016-07-22 at 10 58 54" src="https://cloud.githubusercontent.com/assets/848085/17053783/879ba2ca-4ffb-11e6-9c1c-273c8e743f8b.png">

Instead, what I think we should do is construct a translator instance, pass the acceptLanguage to the translator, get `translator.language` which will be the best match language, best match for our own translations, and then pass that as the locale to moment. 

I did that by applying this patch:

```diff
diff --git a/mailer.js b/mailer.js
index 652927d..74a4a3d 100644
--- a/mailer.js
+++ b/mailer.js
@@ -124,12 +125,14 @@ module.exports = function (log) {
     return ''
   }
 
-  Mailer.prototype._constructLocalTimeString = function (timeZone, locale) {
-    return constructLocalTimeString(timeZone, locale)
+  Mailer.prototype._constructLocalTimeString = function (timeZone, acceptLanguage) {
+    var translator = this.translator(acceptLanguage)
+    return constructLocalTimeString(timeZone, translator.language)
   }
```

With that, the results are correctly translated.